Sources/EpoxyLayoutGroups/Groups/HGroup.swift

```diff
@@ -120,21 +121,26 @@ public final class HGroup: UILayoutGuide, Constrainable, InternalGroup {
     ///                              These will also be used if `forceVerticalAccessibilityLayout` is set to `true`.
     ///                              Individual item alignments will take precedence over this value
     ///   - spacing: the spacing between items of the group
+    ///   - reflowsForAccessibilityTypeSizes: whether or not this group should reflow when DynamicType
+    ///                                         is enabled and is an accessiblity size category
     public init(
       alignment: HGroup.ItemAlignment = .fill,
       accessibilityAlignment: VGroup.ItemAlignment = .leading,
-      spacing: CGFloat = 0)
+      spacing: CGFloat = 0,
+      reflowsForAccessibilityTypeSizes: Bool = true)
     {
       self.alignment = alignment
       self.accessibilityAlignment = accessibilityAlignment
       self.spacing = spacing
+      self.reflowsForAccessibilityTypeSizes = reflowsForAccessibilityTypeSizes
     }
 
     // MARK: Internal
 
     let alignment: HGroup.ItemAlignment
     let accessibilityAlignment: VGroup.ItemAlignment
     let spacing: CGFloat
+    let reflowsForAccessibilityTypeSizes: Bool
```

Comment:
Do you want to include `forceVerticalAccessibilityLayout` as well while you're here?

Reply:
oh yeah, good idea 👍